README.md

```diff
@@ -3,5 +3,18 @@ Alphabetize JSON script
 
 To run, enter the following in the command line:
 ```
-./alphabetize.rb input.json actual.json
+./alphabetize.rb test-input/input.json actual.json
 ```
+
+Then compare the actual result vs the expected:
```

Comment:
Ideally this would be automated by Travis CI or equivalent, but these instructions serve as a stopgap before that's implemented.